src/main/java/com/exasol/errorcodecrawlermavenplugin/crawler/MessageStepReader.java

```diff
@@ -10,22 +11,27 @@
 import spoon.reflect.code.CtInvocation;
 
 /**
- * Reader for invocations of {@link ErrorMessageBuilder#message(String)}.
+ * Reader for invocations of {@link ErrorMessageBuilder#message(String, Object...)}.
  */
 class MessageStepReader implements MessageBuilderStepReader {
-    private static final String SIGNATURE = "message(java.lang.String)";
+    private static final Set<String> SUPPORTED_SIGNATURES = Set.of("message(java.lang.String,java.lang.Object[])",
+            "message(java.lang.String)");
 
     @Override
     public void read(final CtInvocation<?> builderCall, final ErrorMessageDeclaration.Builder errorCodeBuilder,
             final Path projectDirectory) throws InvalidSyntaxException {
         final List<CtExpression<?>> arguments = builderCall.getArguments();
-        assert arguments.size() == 1;
+        assert !arguments.isEmpty();
         final CtExpression<?> messageArgument = arguments.get(0);
-        errorCodeBuilder.prependMessage(new ArgumentReader(SIGNATURE).readStringArgumentValue(messageArgument));
+        final String message = new ArgumentReader(builderCall.getExecutable().getSignature())
+                .readStringArgumentValue(messageArgument);
+        errorCodeBuilder.prependMessage(message);
+        new DirectParameterReader().readInlineParameters(arguments.size() - 1, message, errorCodeBuilder);
     }
 
     @Override
     public boolean canRead(final String className, final String methodSignature) {
-        return className.equals(ErrorMessageBuilder.class.getSimpleName()) && methodSignature.equals(SIGNATURE);
+        return className.equals(ErrorMessageBuilder.class.getSimpleName())
+                && SUPPORTED_SIGNATURES.contains(methodSignature);
```

Comment:
I think there's no test for this condition
`className.equals(ErrorMessageBuilder.class.getSimpleName()) && SUPPORTED_SIGNATURES.contains(methodSignature);`

Reply:
there is. Otherwise, this reader wouldn't be used